buildcomponents.yaml

```diff
@@ -0,0 +1,13 @@
+.migrate:
+  image: kevinrambaud/node-flyway:8-5.1.4-alpine
+  script:
+    - cp -r $CI_PROJECT_DIR/sql/* /flyway/sql/
+    - sh /flyway/flyway -url=jdbc:postgresql://$POSTGRES_SERVER:5432/$POSTGRES_DB -schemas=public -user=$POSTGRES_USER -password=$POSTGRES_PASSWORD migrate
```

Comment:
For B2B projects, we've added a variable to allow us to grant permissions to an environment-specific username in the Flyway script. So if we were going to use this, we'd also want `-placeholders.ENVIRONMENT=$DEPLOYENV` or something like that.

(Order Request is an example – see the [CI file](https://cimpress.githost.io/commerce/b2b/order-request/blob/master/.gitlab-ci.yml) and [related SQL](https://cimpress.githost.io/commerce/b2b/order-request/blob/master/sql/V2__create_index_and_grant_access.sql).)

Reply:
^ yeah, that was awesome.